src/logger.ts

```diff
@@ -1,7 +1,5 @@
 import commonWinston from 'jscommons/dist/winston';
-// tslint:disable-next-line:no-unused
-import * as winston from 'winston'; // tslint:disable-line
-import config from './config';
+import * as winston from 'winston'; import config from './config';
```

Comment:
Guessing this was accidental?
```suggestion
import * as winston from 'winston';
import config from './config';
```

Reply:
Whoops 👀  Yeah, I don't remember if I did this or WebStorm. Thanks 👍